### PR TITLE
String.prototype.lastIndexOf()

### PR DESCRIPTION
**Pull request by data-pup**
Added a binding for `String.prototype.lastIndexOf()`. :smile: Let me know if there's any issues with this!

### CODE REVIEW

**Comment by data-pup** [comment]
Just saw the commit on master regarding fixes for nightly! I can rebase this, expecting it won't pass the CI build.

**Comment by data-pup** [comment]
@sendilkumarn of course! Was fun to learn more a little about wasm-bindgen. I might try and add a few other bindings today :)

**Comment by data-pup** [comment]
Made a minor punctuation edit to the comments above the binding for the sake of consistency.

File: tests/all/js_globals/JsString.rs [hunk 1]
```diff
@@ -261,6 +261,45 @@ fn index_of() {
         .test()
 }
 
+#[test]
+fn last_index_of() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
```
Comment (alexcrichton) [inline tests/all/js_globals/JsString.rs]: I believe to pass tests this'll need to change `proc_macro` to `use_extern_macros`

Comment (data-pup) [reply tests/all/js_globals/JsString.rs]: Ah! Sure thing, thanks for the review.

**Comment by alexcrichton** [comment]
👍

**Comment by data-pup** [comment]
Referencing #275 so people don't duplicate work :)

**Pull request merged**